crates/rooch-types/src/keypair_type.rs

```diff
@@ -1,61 +0,0 @@
-// Copyright (c) RoochNetwork
-// SPDX-License-Identifier: Apache-2.0
-
-use anyhow::Result;
-use clap::ArgEnum;
-#[cfg(any(test, feature = "fuzzing"))]
-use proptest_derive::Arbitrary;
-use schemars::JsonSchema;
-use serde::{Deserialize, Serialize};
-use strum_macros::EnumString;
-
-use crate::error::RoochError;
-
-#[derive(
-    Clone,
-    Copy,
-    Debug,
-    Deserialize,
-    Serialize,
-    Hash,
-    Eq,
-    PartialEq,
-    PartialOrd,
-    Ord,
-    JsonSchema,
-    EnumString,
-    ArgEnum,
-)]
-#[cfg_attr(any(test, feature = "fuzzing"), derive(Arbitrary))]
-#[strum(serialize_all = "lowercase")]
-pub enum KeyPairType {
-    RoochKeyPairType,
-    EthereumKeyPairType,
-}
-
-impl KeyPairType {
-    const ROOCH_KEY_PAIR_TYPE: &str = "rooch";
```

Comment:
Keystore only supports rooch, and will no longer consider supporting other types such as ethereum?

Reply:
Yep.

Reply:
It's not supported for now.